doc/quickstart/find_data.rst

```diff
@@ -593,7 +593,7 @@ Key                  Description                            Default value if not
 ``modeling_realm``   Realm attribute include `atm`, `ice`   No default (needs to be
                      and `oce`                              specified in extra facets or
                                                             recipe if default DRS is used)
-```special_attr``    A special attribute in the filename    No default
+``freq_attribute``   A special attribute in the filename    No default
```

Comment:
As we are changing `special_attr` to freq`_attribute`  we need to edit above as well in lines 576 and 562.  Maybe also add for ocean directories and and files like line 561 and 562 
`'{dataset}/{sub_dataset}/{exp}/{modeling_realm}'`
`'ocean_{freq_attribute}.nc-*'`
and also add to the list of supported variables for ocean (ln 556/557)